Error: Cannot find module 'babel-runtime/regenerator'

### Environment

1.  `react-static -v`: n.a.
2.  `node -v`: v10.6.0
3.  `yarn --version or npm -v`: 1.7.0

### Steps to Reproduce the problem

```
yarn global add react-static@next
react-static create
```

leads to

```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'babel-runtime/regenerator'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/danilo/.config/yarn/global/node_modules/react-static/lib/commands/create.js:8:43)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Module._compile (/Users/danilo/.config/yarn/global/node_modules/pirates/lib/index.js:91:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Object.newLoader [as .js] (/Users/danilo/.config/yarn/global/node_modules/pirates/lib/index.js:96:7)
    at Module.load (internal/modules/cjs/loader.js:599:32)
```

### Expected Behavior

Creating a react-static project.

--- a/babel-preset.js
+++ b/babel-preset.js
@@ -28,7 +28,7 @@ module.exports = () => {
           helpers: false,
           polyfill: false,
           regenerator: true,
-          moduleName: 'babel-runtime',
+          moduleName: '@babel/runtime',
         },
       ],
       r('@babel/plugin-syntax-dynamic-import'),

